terraform/README.adoc

```diff
@@ -82,45 +99,92 @@ overlay
 netlink_diag
 br_netfilter
 ```
+And if you decide to use LVM storage backend then also:
+```
+dm-snapshot
+dm-mirror
+dm_thin_pool
+```
 
-As of writing, lxd is broken by default.  Yesterday, March 17th, a new release
-was made by Ubuntu. This update has not been added to nixpkgs yet. In the
-meantime, you can add this to your configuration.
+LXD config in `/etc/nixos/configuration.nix`:
 
-[source,bash]
+[source,nix]
+----
+  virtualisation.lxd.enable = true;
+  virtualisation.lxd.zfsSupport = false;
+  users.extraGroups.lxd.members = [ "your-user" ];
+  users.extraGroups.lxc.members = [ "your-user" ];
+
+  # Following line needed only if you choose LVM backend for LXC
+  # Following line is a workaround for the issue of lvm tools not being
+  # in the PATH of LXD (https://github.com/NixOS/nixpkgs/issues/31117)
+  systemd.services.lxd.path = with pkgs; [ lvm2 thin-provisioning-tools e2fsprogs ];
+
+  # Needed for kube-proxy pod that crashes if the hashsize is not big enough.
+  # It can't be modified from inside the container even if sys is mounted rw.
+  boot.extraModprobeConfig = ''
+    options nf_conntrack hashsize=393216
+  '';
 ----
- virtualisation.lxd = {
-    enable = true;
-    zfsSupport = false;
-  };
 
- users.users.gila = {
-    isNormalUser = true;
-    extraGroups = [ "wheel" ... "lxd" "lxc" ];
-  };
+After that run `lxd init` and configure it to suit k8s cluster and your
+needs. In particular do not use btrfs or ZFS as a storage pool. Docker's
```

Comment:
This needs to mentioned in the quick start. We say setup lxd correctly but don't say what "correct" is. btrfs is the default choice when running lxd init for the first time

Reply:
I will add that 👍